Src/sym.c

```diff
@@ -159,11 +159,21 @@ trackvar(Lextok *n, Lextok *m)
 				sp->hidden |= 8;
 		}
 		break;
-	case CONST:		// TODO PG - varify for floats
-		if (m->val != 0 && m->val != 1)
-			sp->hidden |= 4;
-		if (m->val < 0 || m->val > 256)
-			sp->hidden |= 8; /* ditto byte-equiv */
+	case CONST:		// TODO PG - not sure if float const are well handled here
```

Comment:
It is my suspicion that this has no sense for floats, as 256.0 is not a special number.... But I am not sure what this code does either. I do not expect any changes, but it would be good to know what it does.

Reply:
unfortunately, the "hidden" field consists of several flags (with different meanings). one of them says "hey, I have hidden attribute". But some of them says "hey, you can cast my value to bit or byte" - that is why I changed that code to handle with floats...